job/server/src/main/java/alluxio/job/plan/migrate/MigrateDefinition.java

```diff
@@ -68,7 +68,7 @@ public MigrateDefinition() {
   }
 
   private void checkMigrateValid(MigrateConfig config, FileSystem fs) throws Exception {
-    AlluxioURI source = new AlluxioURI(config.getSource());
+    AlluxioURI source = new AlluxioURI(config.getFilePath());
```

Comment:
why was this changed? for migrate, there is a source and destination right？

Reply:
previously changed this name so that in `getFilePathForNonBatchConfig` the file paths can be obtained by the same name of `getFilePath()`, which makes them look consistent. I could revert this change if not necessary.

Reply:
is there still a `getDestination` function? I imagine that is still useful?

Reply:
@yuzhu Yeah, getDestination is untouched. If this change causes confusion still, I will revert it.

Reply:
that's fine, please add a comment to `getFilePath` if there isn't one already.

Reply:
I reverted the change of name this time, will revisit it if there are other new needs to change the name.